images/runtime/node/13.9/base.Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
     *) echo "unsupported architecture"; exit 1 ;; \
   esac
 
-ENV NODE_VERSION 12.9.1
```

Comment:
is 12.9 specified by AppService? if so, we should not do this and add a new `13` file